From 35a474d417f9c9e00a66e1460c6ad3818138317d Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Fri, 10 Oct 2025 11:37:10 +0200
Subject: [PATCH] arm64: defconfig: fix display drivers from built-in to module

Update drivers to set all of them as module drivers except DRM &
SIMPLE_DRM.

Change-Id: I08f7d920ad5fd19c62f40bafec08846a8b4cd585
Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/498339
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 .../arm64/configs/fragment-02-defconfig-addons.config | 11 +----------
 1 file changed, 1 insertion(+), 10 deletions(-)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -20,19 +20,10 @@ CONFIG_CRYPTO_DEV_STM32_CRC=m
 CONFIG_CRYPTO_DEV_STM32_CRYP=m
 CONFIG_CRYPTO_DEV_STM32_HASH=m
 CONFIG_DRM=y
-CONFIG_DRM_BRIDGE=y
-CONFIG_DRM_DISPLAY_CONNECTOR=m
 CONFIG_DRM_DW_MIPI_DSI=m
-CONFIG_DRM_I2C_ADV7511=m
-CONFIG_DRM_KMS_HELPER=y
-CONFIG_DRM_MIPI_DSI=y
-CONFIG_DRM_PANEL=y
-CONFIG_DRM_PANEL_BRIDGE=y
-CONFIG_DRM_PANEL_LVDS=m
 CONFIG_DRM_PANEL_RAYDIUM_RM68200=m
-CONFIG_DRM_PANEL_SIMPLE=m
 CONFIG_DRM_SIMPLEDRM=y
-CONFIG_DRM_STM=y
+CONFIG_DRM_STM=m
 CONFIG_DRM_STM_DSI=m
 CONFIG_DRM_STM_LVDS=m
 CONFIG_DWMAC_GENERIC=y
